src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1694,10 +1707,21 @@ static inline fsal_status_t kvsfs_setattrs(struct fsal_obj_handle *obj_hdl,
 		/* If the size does not need to be change, then
 		 * we can simply update the stats associated with the inode.
 		 */
-		rc = cfs_setattr(obj->cfs_fs, &cred,
-				 kvsfs_fh_to_ino(obj->handle),
-				 &stats, flags);
-		result = fsalstat(posix2fsal_error(-rc), -rc);
+		rc = cfs_fh_from_ino(obj->cfs_fs,
```

Comment:
This is a temporary change to have FH from Ino, add common todo comment so that it will be easy to remove that at the end.

Reply:
Done.